controller/api/destination/server_test.go

```diff
@@ -186,6 +186,7 @@ func TestGet(t *testing.T) {
 	})
 }
 
+//TODO: write test where json is fucked and where we won't get a ns token if we split up the string
```

Comment:
May want to clean this comment up a bit 😬

Reply:
That was not supposed to pushed, oops. 😳